### ISSUE TITLE
Submission not working

### ISSUE THREAD

**Issue by Mastermindaxe** [nolanderc/kattis-client#2]
Submission feature is not working atm. I'll try to set it up fresh later this week to see if something with the authentication has changed on my end.

```
Language: Rust
Files:
  - ./src/main.rs
Main Class:
Proceed with the submission? (y/N) y
Submission ID: 123456
Error: expected value at line 1 column 1
```
(Submission ID is a fictional number)

**Comment by nolanderc** [comment]
Thanks for the report!

I can confirm that the submission is broken for me as well. There has probably been some change in how Kattis handles submission. I will attempt to fix this ASAP.

**Comment by nolanderc** [comment]
Kattis used to support an optional GET parameter `only_submission_row` which only grabbed the submission status from the submission page. This was used to tell the state of the submission (which test cases were passed and the final execution time). It seems this parameter is no longer working, which causes this error when trying to fetch the submission status.

Do note that the submission still goes through, it's just the client which fails to read it's status. I'm working on a fix to address this change. As a temporary workaround you can open the submission page in a browser.

**Comment by nolanderc** [comment]
A fix has been implemented in 970febe6c3dbfb3752421ccc4e89f48da56e9ee7. 

The patched version is also available on [crates.io](https://crates.io/crates/kattis) which can be installed by running `cargo install -f kattis`.

**Comment by Mastermindaxe** [comment]
Thank you so much for your efforts and the quick fix! Really love the tool so far :D
